website/js/repl.js

```diff
@@ -75,6 +75,12 @@ var optionsConfig = [
     defaultVal: true,
     description: "Removes Flow type annotations from the output."
   },
+  {
+    type: "boolean",
+    name: "abstractEffectsInAdditionalFunctions",
```

Comment:
Additional -> Pure

Reply:
This is the name of the option in `options.js`.